macros/GroupData.json

```diff
@@ -1018,6 +1037,15 @@
             "properties": [ "Screen.orientation"],
             "events":     []
         },
+        "Screen Wake Lock API": {
+            "overview":   [ "Screen Wake Lock API" ],
+            "interfaces": [ "WakeLock",
+                            "WakeLockSentinel",
+                            "Navigator.wakelock" ],
```

Comment:
Sorry, one last nitpick — `Navigator.wakelock` is a property, not an interface.

Reply:
Sorry of course it is :) should be amended